tests/outputs/correct_pdf_output.txt

```diff
@@ -1,137 +1,43 @@
-STOXX INDEX METHODOLOGY GUIDE CONTENTS
+John Doe
```

Comment:
If I recall correctly, this is a demo file that Rachel was using. However, the output looks completely different from before and are you sure this is the correct output?

Reply:
changed to input file due to model output instability for now